src/pages/login/Login.tsx

```diff
@@ -49,10 +36,13 @@ const Login: FC = () => {
                 </a>
               )}
               {!hasOidc && (
-                <Button hasIcon onClick={openModal} appearance="positive">
-                  <Icon name="security" light />
+                <a
+                  className="p-button--positive has-icon"
+                  href={`${docBaseLink}/howto/oidc`}
+                >
+                  <Icon name="security" />
```

Comment:
```suggestion
                  <Icon name="security" light />
```
Should this stay light theme as before?